tcms/core/contrib/linkreference/views.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# pylint: disable=unused-argument
```

Comment:
Prefix the unused argument with underscore instead of globally disabling this checker. 

Because the offending code is only in 1 place it is better to modify the offence instead of turning a blind eye to possible offences in the future.

Reply:
Useful hint. Will have it in mind in the future.